packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -131,14 +134,18 @@ export const LocalDeviceSettings = (props: LocalDeviceSettingsType): JSX.Element
   /* @conditional-compile-remove(call-readiness) */
   const dropdownProps = usePropsFor(_DevicePermissionDropdown);
 
+  const hasCameras = props.cameras.length > 0 ? true : false;
```

Comment:
nit: the ternaries here redundant, just `props.camera.length > 0` will return true or false